docs/guides-use-metamask.md

```diff
@@ -60,7 +60,7 @@ Why would we need to select other networks? Simply put, before launching a proje
 
 The main reason is that Testnet ETH can be obtained without having to pay real money. This gives developers and the community a chance to iron out any problems before real money is involved.
 
-There are three testnets: **Ropsten**, **Kovan**, and **Rinkleby**.
+There are four testnets: **Ropsten**, **Kovan**, **Rinkleby**, and **Goerli**.
```

Comment:
```suggestion
There are four testnets: **Ropsten**, **Kovan**, **Rinkeby**, and **Goerli**.
```

Although I wonder if we should advertise any testnet other than Rinkeby since we only maintain the Rinkeby deployment.

Reply:
@sohkai up to you. Personally, I think it makes sense to list all testnets since the first half is useful as a general introduction to Metamask (for example, I've linked to it from a Lighthouse validator guide that uses Georli).

Reply:
Sounds good!